File: system-test/sec-tests/typescript/sec_26_security_headers.ts
// SEC-26: Security Response Headers Missing
// Test: TC-CR-032
import express from 'express';

const app = express();
// No helmet or security header middleware
// Missing: CSP, HSTS, X-Frame-Options, X-Content-Type-Options

app.get('/', (req, res) => {
  res.send('Hello');
});

File: system-test/sec-tests/java/Sec26SecurityHeaders.java
// SEC-26: Security Response Headers Missing
// Test: TC-CR-032
package com.example;

import org.springframework.context.annotation.Configuration;
import org.springframework.web.servlet.config.annotation.*;

@Configuration
public class MvcConfig implements WebMvcConfigurer {
    // No CSP, HSTS, X-Frame-Options, X-Content-Type-Options headers configured
}
